### ISSUE TITLE
Using env!("CARGO_PKG_VERSION") in structopt attribute broken in v0.3.10

### ISSUE THREAD

**Issue by forbjok** [TeXitoi/structopt#357]
Using env!("CARGO_PKG_VERSION") (and likely macros in general, although I haven't tested anything other than env!) in the structopt attribute is broken in v0.3.10 and now causes a compile error. It was working fine for a long time until yesterday in v0.3.9.

Downgrading to 0.3.9 also doesn't seem to be an option, as that now produces a number of ```expected an implementor of trait `std::clone::Clone` ```errors.

Compile error:
![ConEmu64_2020-03-01_10-31-02](https://user-images.githubusercontent.com/4687674/75623247-6841d680-5ba8-11ea-92ba-2581d5087996.png)

Minimal reproduction:
[structopt-bug-repro.zip](https://github.com/TeXitoi/structopt/files/4271586/structopt-bug-repro.zip)

A temporary workaround is replacing the direct macro invocations in the attribute with constants containing the result of those macro invocations:
```rust
const VERSION: &str = env!("CARGO_PKG_VERSION");
const AUTHORS: &str = env!("CARGO_PKG_AUTHORS");

#[derive(StructOpt, Debug)]
#[structopt(name = "Repro", version = VERSION, author = AUTHORS)]
struct Opt {
    #[structopt(short = "v", parse(from_occurrences), help = "Verbosity")]
    verbosity: u8,
}
```

**Comment by CreepySkeleton** [comment]
Yeah, the same problem as #354 , fixed by #356 , about to be released.

**Comment by CreepySkeleton** [comment]
The fix published in `0.3.11`
